### PR TITLE
Fix #1559 Implement ExecutionStrategy

### LINKED ISSUE #1559: Adapt latest Terracotta version
This requires work around the `EntityServerService` to adopt the two added methods as the default implementation does not match Ehcache needs.

### PR DESCRIPTION
**Pull request by ljacomet**
* Bump to latest terracotta .9-beta
* ExecutionStrategy first cut
* Make reconfigure invalid for Ehcache

### CODE REVIEW

**Comment by ljacomet** [comment]
I would like this to go after #1555 so that MnM version bumps can be propagated to server without the changes from .9-beta initially.

File: build.gradle [hunk 1]
```diff
@@ -212,7 +212,7 @@ subprojects {
 allprojects {
   tasks.withType(JavaCompile) {
     options.encoding = 'UTF-8'
-    options.compilerArgs += ['-Xlint:unchecked']
+//    options.compilerArgs += ['-Xlint:unchecked']
```
Comment (ljacomet) [inline build.gradle]: That's a mistake

**Comment by ljacomet** [comment]
Can be merged since #1555 is currently blocked by different bugs.

**Comment by AbfrmBlr** [comment]
Can this PR also accommodate following version updates -

•	tc-passthrough-testing 1.0.9.beta2
•	terracotta-core 5.0.9-beta2
•	galvan 1.0.9-beta2

**Comment by ljacomet** [comment]
Updated PR with latest terracotta versions

**Pull request merged**